docs/do-not-merge.md

```diff
@@ -0,0 +1,815 @@
+
+### `lenientAggregatorMerge` deprecated
+
+`lenientAggregatorMerge` property in segment metadata queries has been deprecated. It will be removed in future releases.
+Use `aggregatorMergeStrategy` instead. `aggregatorMergeStrategy` also supports the `latest` and `earliest` strategies in addition to `strict` and `lenient` strategies from `lenientAggregatorMerge`.
+
+[#14560](https://github.com/apache/druid/pull/14560)
+[#14598](https://github.com/apache/druid/pull/14598)
+
+### Broker parallel merge config options
+
+The paths for `druid.processing.merge.pool.*` and `druid.processing.merge.task.*` have been flattened to use `druid.processing.merge.*` instead. The legacy paths for the configs are now deprecated and will be removed in a future release. Migrate your settings to use the new paths because the old paths will be ignored in the future.
+
+[#14695](https://github.com/apache/druid/pull/14695)
+
+### Ingestion options for ARRAY typed columns
+
+Starting with Druid 28.0.0, the MSQ task engine can detect and ingest arrays as ARRAY typed columns when you set the query context parameter `arrayIngestMode` to `array`.
+The `arrayIngestMode` context parameter controls how ARRAY type values are stored in Druid segments.
+
+When you set `arrayIngestMode` to `array` (recommended for SQL compliance), the MSQ task engine stores all ARRAY typed values in [ARRAY typed columns](https://druid.apache.org/docs/latest/querying/arrays) and supports storing both VARCHAR and numeric typed arrays.
+
+For backwards compatibility, `arrayIngestMode` defaults to `mvd`. When `"arrayIngestMode":"mvd"`, Druid only supports VARCHAR typed arrays and stores them as [multi-value string columns](https://druid.apache.org/docs/latest/querying/multi-value-dimensions).
+
+When you set `arrayIngestMode` to `none`, Druid throws an exception when trying to store any type of arrays.
+
+For more information on how to ingest `ARRAY` typed columns with SQL-based ingestion, see [SQL data types](https://druid.apache.org/docs/latest/querying/sql-data-types#arrays) and [Array columns](https://druid.apache.org/docs/latest/querying/arrays).
+
+## Incompatible changes
+
+### Removed Hadoop 2
+
+Support for Hadoop 2 has been removed.
+Migrate to SQL-based ingestion or JSON-based batch ingestion if you are using Hadoop 2.x for ingestion today.
+If migrating to Druid's built-in ingestion is not possible, you must upgrade your Hadoop infrastructure to 3.x+ before upgrading to Druid 28.0.0.
+
+[#14763](https://github.com/apache/druid/pull/14763)
+
+### Removed GroupBy v1 
+
+The GroupBy v1 engine has been removed. Use the GroupBy v2 engine instead.  
+If you are using GroupBy v1 in JSON-based batch queries, you must change your queries before upgrading. Otherwise, your queries will fail.
+For more information, see [GroupBy queries](https://druid.apache.org/docs/latest/querying/groupbyquery.html).
```

Comment:
hm, i don't think the queries fail, just the option is ignored completely and grouping always uses v2, but i can double check...

Reply:
@clintropolis Can you please double check? I can update the message based on your findings.

Reply:
yeah, it is just ignored

Reply:
Updated